bench/workgen/runner/example_dynamic_tables.py

```diff
@@ -106,13 +117,46 @@ def create(session, workload, table_config):
 workload_thread = ThreadWithReturnValue(target=workload.run, args=([connection]))
 workload_thread.start()
 
-# Create tables while the workload is running.
+# Create and drop tables while the workload is running.
+tables_to_delete = []
 while workload_thread.is_alive():
     create(session, workload, table_config)
+
+    # Select a random table to drop.
+    if len(tables):
+        idx = random.randint(0, len(tables) - 1)
+        uri = tables[idx]
+        # Mark it for deletion.
+        drop(workload, uri)
+
+    # Trigger the Workgen garbage collection.
+    tables_to_delete += workload.garbage_collection()
```

Comment:
@ruby-oujo, here we are retrieving the tables that have been deleted in workgen. Note the `+=` as we may already have deleted tables in the past that we have not been able to process here at the Python level.